src/main/java/com/atlassian/db/replica/api/AuroraConnectionDetails.java

```diff
@@ -1,44 +0,0 @@
-package com.atlassian.db.replica.api;
-
-public final class AuroraConnectionDetails {
```

Comment:
Removing API class breaks compatibility.

Reply:
https://wiki.eclipse.org/Evolving_Java-based_APIs_2#Evolving_API_Classes

Reply:
restored the class to preserve compatibility